packages/api/start/hooks.js

```diff
@@ -29,4 +29,14 @@ hooks.after.providersBooted(() => {
 	Object.values(CustomValidators).forEach((validator) => {
 		Validator.extend([validator.name], validator);
 	});
+
+	const Env = use('Env');
+	const View = use('View');
+
+	View.global('APP_URL', () => {
+		return Env.get('APP_URL');
+	});
+	View.global('WEB_URL', () => {
+		return Env.get('WEB_URL');
+	});
```

Comment:
Não é uma boa prática usar o env diretamente mas sim sempre acessá-lo pela camada de configuração
```suggestion
	View.global('APP_URL', () => Config.get('app.APP_URL'));
	/** ... */
```
Também suponho que possamos passar como parâmetro a string resultante do `Config.get()` ao invés da função anônima. Assim, acessamos o valor no Edge como variável e não como função 👇 
```js
// hooks.js
View.global('APP_URL', Config.get('app.APP_URL'));

// file.edge
<a href="{{APP_URL}}/login">Login</a>
```